Dockerfile

```diff
@@ -21,6 +21,15 @@ RUN apt-get update && \
 VOLUME /build
 WORKDIR /build
 
+RUN mkdir -p Client Server Shared utils vendor
+
+COPY Client Client
+COPY Server Server
+COPY Shared Shared
+COPY utils utils
+COPY vendor vendor
+COPY ./* .
+

```

Comment:
You should use `-v' parameter in `docker run`  to mount these folders into a container

Reply:
How would it work?

Reply:
You should mount mta repository (or an empty folder) to `/build` folder in a container. `docker-entrypoint.sh` runs inside this folder.